### ISSUE TITLE
Docker version

### ISSUE THREAD

**Issue by cocomine** [jagrosh/MusicBot#188]
This is not issues
I just talk everyone, If you want a Docker version You can get it here
https://hub.docker.com/r/cocomine/jmusicbot/

**Comment by spotlightishere** [comment]
Must admit, very confused on how one might use this.

 - Why use the `webupd8` ppa manually? (i.e, why not pull from Oracle's images?)
 - Why not `ADD` the jar separately for versioning?
 - Does one bind their `config.txt` to `/` or in volume `/bot`?

A PR with the Dockerfile (#22) would help as well, potentially, especially if built from scratch. (assuming jag'd be okay with it, as well)

**Comment by cocomine** [comment]
>Does one bind their config.txt to / or in volume /bot?
Because some programs are not too complete for the terminal display(e.x. Synology NAS docker)

>Why not ADD the jar separately for versioning?
I don't have much time to maintain. So I used environment variables (VAR) to determine the version. Users can fill in by themselves. Then the file is download directly from github

>Why use the webupd8 ppa manually? (i.e, why not pull from Oracle's images?)
I didn't take it into consideration when I built it. And I use this method to install java on my own computer. After that, chances are that you will adopt your suggestion.
